src/absil/ilread.fsi

```diff
@@ -64,23 +64,28 @@ type internal ILReaderOptions =
      /// and from which we can read the metadata. Only used when metadataOnly=true.
      tryGetMetadataSnapshot: ILReaderTryGetMetadataSnapshot }
 
+
 /// Represents a reader of the metadata of a .NET binary.  May also give some values (e.g. IL code) from the PE file
 /// if it was provided.
-[<Sealed>]
-type internal ILModuleReader =
-    member ILModuleDef : ILModuleDef
-    member ILAssemblyRefs : ILAssemblyRef list
+type IModuleReader =
+    abstract ILModuleDef: ILModuleDef
+    abstract ILAssemblyRefs: ILAssemblyRef list
     
     /// ILModuleReader objects only need to be explicitly disposed if memory mapping is used, i.e. reduceMemoryUsage = false
-    interface System.IDisposable
+    inherit System.IDisposable
+
+
+[<Sealed>]
+type ILModuleReader =
```

Comment:
Just use the name `ILModuleReader` for the interface and hide this concrete implementation type please (I don't think it needs to be public?)

Re `I` naming - for example we already use `ImportMap` in the compiler even though it's an interface type.